Mobile-Rounds.ViewModels/Shared/Home/HomePageViewModel.cs

```diff
@@ -35,6 +35,7 @@ public HomePageViewModel()
         {
             this.Sync = new SyncCommand();
             this.StartRound = new StartRoundCommand();
+            this.Crumbs.Add(new Controls.BreadcrumbItemModel("HELLO WORLD", new GoHomeCommand()));
```

Comment:
@BMVW-Capstone/students This is how you can add a breadcrumb item into your view.